### PR TITLE
Add styleguide.md

### PR DESCRIPTION
**Pull request by astroash**
Relates #788 

work in progress

### CODE REVIEW

File: curriculum-planning/style-guide.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Week research-afternoon.md
+research-afternoon.md should include:
+- 4 heading for each topic to be researched
+- a set of questions each research topic aims to answer
+
+### Week resources.md
+resources.md is a collection of links to extra material students can use to prepare/revise the learning of the week. They should:
+- be separated under the same subsections as week LO's
+
+### Week slides.md
```
Comment (m4v15) [inline curriculum-planning/style-guide.md]: Maybe have something here that the slides should work by themselves, and are not 100% necessary for a mentor or whoever wrote it being around to deliver it (maintain self/peer-led learning aspect)

File: curriculum-planning/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+> How to test an app that uses a database :ok_hand:
+> How to setup an environment variable for a test database :warning: _too specific to workshop_
+>
+
+### Week Project.md
+Project's are a chance for students to apply the knowledge they've acquired by building a web app from scratch. They should include:
+- a link to the week's LO's
+- a description of the aims of the app
+- technical requirements
+- usability requirements in the form of User Stories
```
Comment (m4v15) [inline curriculum-planning/style-guide.md]: Do they all need user stories? I think towards the latter stages it's good for the students to have to think up user stories themselves

File: curriculum-planning/style-guide.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Creation of New Resources
+When making a new workshop/morning challenge:
+1. Create a new repo within the foundersandcoders GitHub organisation
+2. Follow the repo naming convention - prepend the name with:
+- ws- for workshops
+- mc- for morning challenges
+- sd- for repo based slides
+
+3. Make a PR to the master-reference, editing the resource index to add a link to your repo
```
Comment (m4v15) [inline curriculum-planning/style-guide.md]: 4. Assign WF or MRM to review + any other relevant parties

File: curriculum-planning/style-guide.md (hunk 2 above)
Comment (astroash) [reply curriculum-planning/style-guide.md]: Yeah, makes sense. Do you think this should be explicitly mentioned in project.md's eg in week 4 onwards state as a requirement "You must create your own user stories"?

Comment (m4v15) [reply curriculum-planning/style-guide.md]: yeah that sounds good 👍

**Comment by eliascodes** [comment]
@astroash see [my comment](https://github.com/foundersandcoders/master-reference/issues/788#issuecomment-354436535)

**Comment by eliascodes** [comment]
@astroash did the creating/maintaining re-structuring we discussed and added a few points. Let me know what you think.

File: curriculum-planning/style-guide.md [hunk 4]
```diff
@@ -1,87 +1,109 @@
...
-### Week Project.md
-Project's are a chance for students to apply the knowledge they've acquired by building a web app from scratch. They should include:
-- a link to the week's LO's
+### project.md
+Projects are a chance for students to apply the knowledge they've acquired by building a web app from scratch. They should include:
+- a link to the week's learning objectives
 - a description of the aims of the app
 - technical requirements
-- usability requirements in the form of user stories for weeks 1-4. Md should state that students create their own user stories weeks 5-8.
+- user requirements in the form of user stories for weeks 1-4. Instruction that students should create their own user stories weeks 5-8.
```
Comment (astroash) [inline curriculum-planning/style-guide.md]: Does this sentence make sense?

Comment (eliascodes) [reply curriculum-planning/style-guide.md]: Meant to be a run-on from the colon (i.e. should include...instruction that), but that's probably clearer if it's moved to its own bullet?

Comment (astroash) [reply curriculum-planning/style-guide.md]: Yeah, I'd stick it as another bullet point.  Other than that, this is great.

**Comment by astroash** [comment]
@eliascodes great emoji game :game_die:

**Comment by m4v15** [comment]
@eliascodes I was initially just being annoying and adding a single 's' but then @astroash  mentioned how maybe we want to be less prescriptive on the number of research topics, so I've changed it again

**Pull request merged**